togetherly/lib/views/widgets/chore_list.dart

```diff
@@ -4,39 +4,15 @@ import 'package:togetherly/themes.dart';
 import 'package:togetherly/views/widgets/chore_item.dart';
 
 class ChoreList extends StatefulWidget {
-  const ChoreList({super.key, required this.title});
+  const ChoreList({super.key, required this.title, required this.choreList});
   final String title;
+  final List<Chore> choreList;
 
   @override
   State<ChoreList> createState() => _ChoreListState();
 }
 
 class _ChoreListState extends State<ChoreList> {
-  List<Chore> chores = [
-    Chore(
-      title: 'Do the dishes',
-      dueDate: DateTime(2024, 2, 15),
-      points: 15,
-      assignedChildId: 0,
-      isShared: false,
-    ),
-    Chore(
-      title: 'Pick up the living room',
-      dueDate: DateTime(2024, 2, 14),
-      points: 10,
-      status: ChoreStatus.pending,
-      assignedChildId: 0,
-      isShared: false,
-    ),
-    Chore(
-      title: 'Walk the dog',
-      dueDate: DateTime(2024, 2, 14),
-      points: 20,
-      status: ChoreStatus.completed,
-      assignedChildId: 0,
-      isShared: false,
-    ),
-  ];
```

Comment:
Will this widget need to maintain state in the future? Or can it be made stateless?

Reply:
I assumed that any widget consuming a provider would need to be stateful, but looking at that now I see that the widget containing the AppProviders consumer is stateless, so I can change this one to be stateless as well.